.vscode/settings.json

```diff
@@ -0,0 +1,45 @@
+{
```

Comment:
If you're committing this, you should also add the extensions you need to the settings file (prettier and eslint dont work without the extensions for example)

Reply:
add prettier & eslint into extensions.json

The html, json, javascript, typescript are built in feature in VScode